crt_portal/cts_forms/management/commands/flag_repeat_writers.py

```diff
@@ -0,0 +1,53 @@
+from django.db import connection
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Flag reports that were submitted by repeat writers."
+
+    def handle(self, *args, **options):
+        repeat_summary_sql = """
+        WITH repeat_summary AS (SELECT violation_summary, COUNT(*)
+        FROM cts_forms_report
+        GROUP BY violation_summary
+        HAVING count(*) > 50 )
+        UPDATE cts_forms_report
+        SET by_repeat_writer = true
+        WHERE id IN
+        (SELECT report.id
+        FROM cts_forms_report report
+        JOIN repeat_summary
+        ON report.violation_summary = repeat_summary.violation_summary
+        ORDER BY report.violation_summary);
+        """
```

Comment:
This is cool! I think I'm gonna need you to walk me through this query though...

Reply:
Sure thing 👍🏻

Reply:
Also, here is some relevant documentation that might prove helpful. https://www.postgresql.org/docs/current/queries-with.html